doc/nrf/app_dev/device_guides/fem/21540ek_dev_guide.rst

```diff
@@ -52,14 +52,20 @@ For example:
 .. parsed-literal::
    :class: highlight
 
-   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dmultiprotocol_rpmsg_SHIELD=nrf21540ek
+   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dipc_radio_SHIELD=nrf21540ek
```

Comment:
do we put `--sysbuild` option ?

Reply:
No: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/config_and_build/config_and_build_system.html#sysbuild-enabled-ncs

Reply:
It is not needed, sysbuild is enabled by default